rcl_yaml_param_parser/QUALITY_DECLARATION.md

```diff
@@ -22,6 +22,7 @@ The current version can be found in its [package.xml](package.xml), and its chan
 All symbols in the installed headers are considered part of the public API.
 
 All installed headers are in the `include` directory of the package, headers in any other folders are not installed and considered private.
+Headers in the `include/rcl_yaml_param_parser/impl` directory should also be considered private.
```

Comment:
If they're private why do they need to be in `include`? This is C, not C++, there should be no need to expose a symbol but also consider it to be private, I think. The only reason to do that in C++ is for templates which use functions and inline implementations in headers. Since neither of those are the case for C, you should be able to just move these into `src`, right?

Reply:
Moved headers into `src` directory

Reply:
Err, looks like functions need to be labeled `RCL_YAML_PARAM_PARSER_PUBLIC` not LOCAL for it to work with the new unit tests.

Reply:
They can be made visible without being in the include folder.

Reply:
Oh yah totally. In response to this comment, I moved the headers to source and switched it to `LOCAL`, but the followup force push was just me switching them back to `PUBLIC`. Headers are still in `src`.